cli/tests/017_import_redirect_info.out

```diff
@@ -1,6 +1,6 @@
 local: [WILDCARD]017_import_redirect.ts
 type: TypeScript
-deps: 1 unique (total [WILDCARD]B)
+deps: 2 unique (total [WILDCARD]B)
```

Comment:
What's the reason for this change? It seems redirects are now counted as unique deps?

Reply:
hmmmm... yeah, I guess that isn't right... I will try to fix that...  need to count unique `Module` module slots.